src/style.css

```diff
@@ -188,10 +188,118 @@ footer .f-top {
   border-top: 1px solid rgba(0, 0, 0, 0.473);
   padding: 1.5em 0.5em;
   font-size: x-small;
+  text-align: center;
 }
 
 footer .f-bottom {
   background: #0000004a;
   width: 100%;
   height: 1.5em;
 }
+
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+  width: 90vw;
+  min-height: 565px;
+  border: 2px solid rgb(223, 17, 113);
+  animation: 1s drop;
+  display: none;
+  padding: 0 8px;
+}
+
+.close-button {
+  position: absolute;
+  cursor: pointer;
+  outline: none;
+  margin-right: 4px;
+  background-color: #fff;
+  color: #67798e;
+  border: none;
+  font-size: 30px;
+  font-weight: 600;
+  top: 0;
+  right: 0;
+}
+
+.add-comm {
+  display: flex;
+  justify-content: center;
+}
+
+.inputDiv {
+  margin: 0 300px;
+}
+
+#name {
+  width: 200px;
+  color: #67798e;
+  margin-bottom: 16px;
+  padding: 5px;
+}
+
+#text {
+  width: 200px;
+  color: #67798e;
+  margin-bottom: 16px;
+}
+
+.submit-btn {
+  padding: 5px 18px;
+  font-size: 15px;
+  margin-top: 10px;
+  color: red;
+}
+
+.pop {
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  position: relative;
+  top: 22px;
+  padding: 20px;
+}
+
+.popIt {
+  position: fixed;
+  top: 10px;
+  max-height: 100vh;
+  overflow: auto;
+  width: 100%;
+}
+
+.popup_image {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.pop_image {
+  width: 200px;
+  margin: auto;
+}
+
+.instructions {
+  height: 100px;
+  margin-top: 20px;
+  width: 700px;
+  overflow: auto;
+
+  /* background: #eeb3b3; */
+  font-size: 14px;
+}
+
+.pop_meal {
+  font-size: 20px;
+}
+
+.flex {
+  display: flex;
+}
+
+.column {
+  flex-direction: column;
```

Comment:
kindly note the styles in here project a different view from the expected Wireframe.
```suggestion

.modal {
  position: fixed;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  z-index: 10;
  width: 90vw;
  min-height: 565px;
  border: 2px solid rgb(223, 17, 113);
  animation: 1s drop;
  display: none;
  padding: 0 8px;
}

.close-button {
  position: absolute;
  cursor: pointer;
  outline: none;
  margin-right: 4px;
  background-color: #fff;
  color: #67798e;
  border: none;
  font-size: 30px;
  font-weight: 600;
  top: 0;
  right: 0;
}

.add-comm {
  display: flex;
  justify-content: center;
}

.inputDiv {
  margin: 0 300px;
}

#name {
  width: 200px;
  color: #67798e;
  margin-bottom: 16px;
  padding: 5px;
}

#text {
  width: 200px;
  color: #67798e;
  margin-bottom: 16px;
}

.submit-btn {
  padding: 5px 18px;
  font-size: 15px;
  margin-top: 10px;
  color: red;
}

.pop {
  justify-content: center;
  align-items: center;
  background: #fff;
  position: relative;
  top: 22px;
  padding: 20px;
}

.popIt {
  position: fixed;
  top: 10px;
  max-height: 100vh;
  overflow: auto;
  width: 100%;
}

.popup_image {
  width: 100%;
  display: flex;
  justify-content: center;
}

.pop_image {
  width: 200px;
  margin: auto;
}

.instructions {
  height: 100px;
  margin-top: 20px;
  width: 700px;
  overflow: auto;

  /* background: #eeb3b3; */
  font-size: 14px;
}

.pop_meal {
  font-size: 20px;
}

.flex {
  display: flex;
}

.column {
  flex-direction: column;
```